pyproject.toml

```diff
@@ -2,3 +2,6 @@
 testpaths = [
     "pycromanager/test",
 ]
+
+[tool.black]
+line-length = 100
```

Comment:
I think that lines in this repo tend to be longer than the black default 88 so i used 100. but ultimately this was a fairly arbitrary choice

Reply:
I have a wide monitor :)